infra/base-images/base-runner/coverage

```diff
@@ -22,16 +22,19 @@ else
   FUZZ_TARGETS="$(find . -maxdepth 1 -type f -executable)"
 fi
 
+DUMPS_DIR="$OUT/dumps"
+rm -rf $DUMPS_DIR && mkdir -p $DUMPS_DIR
```

Comment:
Instead of repeating rm -rf

we should define the vars first
then use for loop to delete and recreate for all the 4 vars.
for i in $DUMPS_DIR $...

Reply:
Done